setup.py

```diff
@@ -31,6 +31,7 @@
     "audio": ["miniaudio"],
     "gcp": ["google-cloud-storage", "google-auth", "google-auth-oauthlib"],
     "video": ["av"],
+    "visualizer": ["IPython", "flask"],
```

Comment:
you need to add these to common.txt too

Reply:
I dont think we need the `IPython` dependency.